Assets/ProceduralPipeline/Nodes/Buildings/GenerateOSMBuildingClassesNode.cs

```diff
@@ -55,6 +55,11 @@ public OSMBuildingData(List<Vector3> footprint, OSMTags tags)
         MakeRelative();
 		SetHeightAndLevels(tags.height, tags.levels);
 		SetElevation(footprint);
+
+		if (tags.building == "museum" || tags.tourism == "museum")
```

Comment:
Could be a cleaner way of doing this?

Reply:
Could you please take another look when it is convenient? Thanks